<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <title>Welcome to FitHiChIP! &#8212; FitHiChIP 0.1 documentation</title>
    <link rel="stylesheet" href="static/bootstrap-sphinx.css" type="text/css" />
    <link rel="stylesheet" href="static/pygments.css" type="text/css" />
    <script type="text/javascript" id="documentation_options" data-url_root="./" src="static/documentation_options.js"></script>
    <script type="text/javascript" src="static/jquery.js"></script>
    <script type="text/javascript" src="static/underscore.js"></script>
    <script type="text/javascript" src="static/doctools.js"></script>
    <script type="text/javascript" src="static/language_data.js"></script>
    <script type="text/javascript" src="static/js/jquery-1.11.0.min.js"></script>
    <script type="text/javascript" src="static/js/jquery-fix.js"></script>
    <script type="text/javascript" src="static/bootstrap-3.3.7/js/bootstrap.min.js"></script>
    <script type="text/javascript" src="static/bootstrap-sphinx.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Installation" href="usage/installation.html" />
<meta charset='utf-8'>
<meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
<meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1'>
<meta name="apple-mobile-web-app-capable" content="yes">

  </head><body>

  <div id="navbar" class="navbar navbar-default navbar-fixed-top">
    <div class="container">
      <div class="navbar-header">
        <!-- .btn-navbar is used as the toggle for collapsed navbar content -->
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".nav-collapse">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a class="navbar-brand" href="#">
          FitHiChIP</a>
        <span class="navbar-text navbar-version pull-left"><b></b></span>
      </div>

        <div class="collapse navbar-collapse nav-collapse">
          <ul class="nav navbar-nav">
            
            
              <li class="dropdown globaltoc-container">
  <a role="button"
     id="dLabelGlobalToc"
     data-toggle="dropdown"
     data-target="#"
     href="#">Site <b class="caret"></b></a>
  <ul class="dropdown-menu globaltoc"
      role="menu"
      aria-labelledby="dLabelGlobalToc"><ul>
<li class="toctree-l1"><a class="reference internal" href="usage/installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/quickstart.html">Quick start</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/configuration.html">Setting up configuration file</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/output.html">Details of FitHiChIP outputs</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/optional.html">Setting up optional parameters</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/DiffLoops.html">Differential analysis of HiChIP loops</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/Utilities.html">Various utility scripts / functions</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/FAQ.html">Frequently asked question</a></li>
</ul>
</ul>
</li>
              
                <li class="dropdown">
  <a role="button"
     id="dLabelLocalToc"
     data-toggle="dropdown"
     data-target="#"
     href="#">Page <b class="caret"></b></a>
  <ul class="dropdown-menu localtoc"
      role="menu"
      aria-labelledby="dLabelLocalToc"><ul>
<li><a class="reference internal" href="#">Welcome to FitHiChIP!</a><ul>
<li><a class="reference internal" href="#citation">Citation</a></li>
<li><a class="reference internal" href="#contact">Contact</a></li>
</ul>
</li>
</ul>
</ul>
</li>
              
            
            
              
                
  <li>
    <a href="usage/installation.html" title="Next Chapter: Installation"><span class="glyphicon glyphicon-chevron-right visible-sm"></span><span class="hidden-sm hidden-tablet">Installation &raquo;</span>
    </a>
  </li>
              
            
            
            
            
              <li class="hidden-sm">
<div id="sourcelink">
  <a href="sources/index.rst.txt"
     rel="nofollow">Source</a>
</div></li>
            
          </ul>

          
            
<form class="navbar-form navbar-right" action="search.html" method="get">
 <div class="form-group">
  <input type="text" name="q" class="form-control" placeholder="Search" />
 </div>
  <input type="hidden" name="check_keywords" value="yes" />
  <input type="hidden" name="area" value="default" />
</form>
          
        </div>
    </div>
  </div>

<div class="container">
  <div class="row">
    <div class="col-md-12 content">
      
  <div class="section" id="welcome-to-fithichip">
<h1>Welcome to FitHiChIP!<a class="headerlink" href="#welcome-to-fithichip" title="Permalink to this headline">¶</a></h1>
<p><strong>Developers</strong>: Sourya Bhattacharyya, Ferhat Ay</p>
<p><strong>Institution</strong>: La Jolla Institute for Allergy and Immunology, La Jolla, CA 92037, USA</p>
<p>FitHiChIP analyzes HiChIP / PLAC-seq data and derives the statistical significant CIS interactions, given a specific distance range (default = 20 Kb to 2 Mb: can be user defined).</p>
<p>Input of FitHiChIP is the valid CIS read pairs generated from the HiC-Pro pipeline (Servant et. al. 2015), the most commonly used Hi-C data processing pipeline. These read pairs are analyzed to find the statistically significant loops within the given distance range. FitHiChIP uses the distance decay model of the interaction count, proposed in the method FitHiC (Ay and Noble, 2014) and also applies a bias specific regression model to find the statistical significant interactions.</p>
<p>Details of execution commands and parameters are provided in the below mentioned links:</p>
<div class="toctree-wrapper compound">
<ul>
<li class="toctree-l1"><a class="reference internal" href="usage/installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/quickstart.html">Quick start</a><ul>
<li class="toctree-l2"><a class="reference internal" href="usage/quickstart.html#downloading-the-test-data">Downloading the test data</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/quickstart.html#configuration-parameters-according-to-the-test-data">Configuration parameters according to the test data</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/quickstart.html#sample-logs-from-the-console-corresponding-to-the-testdata">Sample logs from the console, corresponding to the TestData</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/quickstart.html#details-of-output-files">Details of output files</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="usage/configuration.html">Setting up configuration file</a><ul>
<li class="toctree-l2"><a class="reference internal" href="usage/configuration.html#a-input-file-name-related-parameters">A) Input File name related parameters</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/configuration.html#b-reference-genome-related-parameters">B) Reference genome related parameters</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/configuration.html#c-generic-parameters-for-profiling-interactions">C) Generic parameters for profiling interactions</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/configuration.html#d-parameters-for-significant-loop-calling">D) Parameters for significant loop calling</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/configuration.html#e-bias-correction-related-parameters">E) Bias correction related parameters</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/configuration.html#f-merging-adjacent-interactions">F) Merging adjacent interactions</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/configuration.html#g-miscellaneous-parameters">G) Miscellaneous parameters</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="usage/output.html">Details of FitHiChIP outputs</a><ul>
<li class="toctree-l2"><a class="reference internal" href="usage/output.html#summary-html-file">Summary HTML file</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/output.html#details-of-output-files">Details of output files</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/output.html#visualizing-significant-interactions-in-washu-epigenome-browser">Visualizing significant interactions in WashU epigenome browser</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/output.html#visualizing-significant-interactions-in-ucsc-genome-browser">Visualizing significant interactions in UCSC genome browser</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="usage/optional.html">Setting up optional parameters</a><ul>
<li class="toctree-l2"><a class="reference internal" href="usage/optional.html#downloading-reference-genome-fasta-sequence">Downloading reference genome fasta sequence</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/optional.html#downloading-the-mappability-file-corresponding-to-the-reference-genome">Downloading the mappability file (corresponding to the reference genome)</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="usage/DiffLoops.html">Differential analysis of HiChIP loops</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage/Utilities.html">Various utility scripts / functions</a><ul>
<li class="toctree-l2"><a class="reference internal" href="usage/Utilities.html#generating-contact-matrices-of-different-resolution-bin-size">1. generating contact matrices of different resolution (bin size)</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/Utilities.html#inferring-peaks-from-hichip-data-for-use-in-the-hichip-pipeline">2. inferring peaks from HiChIP data (for use in the HiChIP pipeline)</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/Utilities.html#merging-a-set-of-chip-seq-alignment-files">3. merging a set of ChIP-seq alignment files</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/Utilities.html#producing-fithichip-loops-with-different-fdr-thresholds">4. Producing FitHiChIP loops with different FDR thresholds</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/Utilities.html#applying-merge-filtering-on-any-set-of-significant-loops-generated-by-fithichip-or-any-other-loop-calling-method">5. Applying merge filtering on any set of significant loops (generated by FitHiChIP or any other loop calling method)</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/Utilities.html#visualization-of-the-significant-loops-in-washu-epigenome-browser">6. Visualization of the significant loops in WashU epigenome browser</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="usage/FAQ.html">Frequently asked question</a><ul>
<li class="toctree-l2"><a class="reference internal" href="usage/FAQ.html#how-to-install-hic-pro-in-the-system">1. How to install HiC-pro in the system?</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/FAQ.html#how-to-specify-the-hic-pro-base-installation-directory-in-the-fithichip-configuration-file">2. How to specify the HiC-pro base installation directory in the FitHiChIP configuration file?</a></li>
<li class="toctree-l2"><a class="reference internal" href="usage/FAQ.html#how-to-visualize-the-fithichip-loops-in-genome-browser">3. How to visualize the FitHiChIP loops in genome browser?</a></li>
</ul>
</li>
</ul>
</div>
<div class="section" id="citation">
<h2>Citation<a class="headerlink" href="#citation" title="Permalink to this headline">¶</a></h2>
<p>If you are using FitHiChIP, please cite:</p>
<p>Sourya Bhattacharyya, Vivek Chandra, Pandurangan Vijayanand, and Ferhat Ay, <em>FitHiChIP: Identification of significant chromatin contacts from HiChIP data</em>, preprint at &lt;<a class="reference external" href="https://www.biorxiv.org/content/early/2018/09/10/412833">https://www.biorxiv.org/content/early/2018/09/10/412833</a>&gt;, DOI: &lt;<a class="reference external" href="https://doi.org/10.1101/412833">https://doi.org/10.1101/412833</a>&gt;</p>
</div>
<div class="section" id="contact">
<h2>Contact<a class="headerlink" href="#contact" title="Permalink to this headline">¶</a></h2>
<ul class="simple">
<li>Sourya Bhattacharyya &lt;<a class="reference external" href="mailto:sourya&#37;&#52;&#48;lji&#46;org">sourya<span>&#64;</span>lji<span>&#46;</span>org</a>&gt;</li>
<li>Ferhat Ay &lt;<a class="reference external" href="mailto:ferhatay&#37;&#52;&#48;lji&#46;org">ferhatay<span>&#64;</span>lji<span>&#46;</span>org</a>&gt;</li>
</ul>
</div>
</div>


    </div>
      
  </div>
</div>
<footer class="footer">
  <div class="container">
    <p class="pull-right">
      <a href="#">Back to top</a>
      
    </p>
    <p>
        &copy; Copyright 2019, Sourya Bhattacharyya, Ferhat Ay.<br/>
      Created using <a href="http://sphinx-doc.org/">Sphinx</a> 1.8.3.<br/>
    </p>
  </div>
</footer>
  </body>
</html>